packages/ubuntu_desktop_installer/lib/l10n/app_es.arb

```diff
@@ -0,0 +1,113 @@
+  "appTitle": "Instalador de Ubuntu para escritorio",
+
+  "backButtonText": "Volver",
+  "continueButtonText": "Continuar",
+  "restartButtonText": "Reiniciar",
+
+  "welcome": "Bienvenido",
+
+  "tryOrInstallPageTitle": "Probar o instalar",
+  "repairInstallation": "Reparar instalación",
+  "repairInstallationDescription": "Al reparar se reinstalarán todos los programas instalados sin tocar los documentos ni la configuración.",
+  "tryUbuntu": "Probar Ubuntu",
+  "tryUbuntuDescription": "Puede probar Ubuntu sin hacer ningún cambio en su equipo.",
+  "installUbuntu": "Instalar Ubuntu",
+  "installUbuntuDescription": "Instalar Ubuntu junto con (o reemplazando) su sistema operativo actual. No debería tardar mucho.",
+  "releaseNotesLabel": "Aquí puede leer las <a href=\"{url}\">notas de lanzamiento</a>.",
+  "@releaseNotesLabel": {
+    "type": "text",
+    "placeholders": {
+      "url": {}
+    }
+  },
+
+  "turnOffRST": "Apagar RST",
+  "turnOffRSTDescription": "Este equipo utiliza Intel RST (Rapid Storage Technology). Es necesario apagar RST en Windows antes de instalar Ubuntu.",
+  "instructionsForRST": "Para seguir las instrucciones, abra esta página en un dispositivo móvil u otro dispositivo: <a href=\"https://{url}\">{url}</a>",
+  "@instructionsForRST": {
+    "type": "text",
+    "placeholders": {
+      "url": {}
+    }
+  },
+
+  "keyboardLayoutPageTitle": "Distribución del teclado",
+  "chooseYourKeyboardLayout": "Elija la distribución del teclado:",
+  "typeToTest": "Escriba aquí para probar el teclado",
+  "detectLayout": "Detectar la distribución del teclado",
+
+  "updatesOtherSoftwarePageTitle": "Actualizaciones y otro software",
+  "updatesOtherSoftwarePageDescription": "¿Qué aplicaciones le gustaría instalar para comenzar?",
+  "normalInstallationTitle": "Instalación normal",
+  "normalInstallationSubtitle": "Navegador web, utilidades, aplicaciones de oficina, juegos y reproductores de contenido.",
+  "minimalInstallationTitle": "Instalación mínima",
+  "minimalInstallationSubtitle": "Navegador web y utilidades básicas.",
+  "otherOptions": "Otras opciones",
+  "installThirdPartyTitle": "Instalar software de terceros para controladores de gráficos y WiFi, así como formatos de contenido adicionales",
+  "installThirdPartySubtitle": "Este software está sujeto a términos de licencia incluidos en su documentación. Algunos son propietarios.",
+
+  "allocateDiskSpace": "Asignar espacio de disco",
+  "startInstallingButtonText": "Comenzar la instalación",
+  "diskHeadersDevice": "Dispositivo",
+  "diskHeadersType": "Tipo",
+  "diskHeadersMountPoint": "Punto de montaje",
+  "diskHeadersSize": "Tamaño",
+  "diskHeadersUsed": "Usado",
+  "diskHeadersSystem": "Sistema",
+  "diskHeadersFormat": "Formato",
+
+  "writeChangesToDisk": "Escribir cambios en el disco",
+  "writeChangesFallbackSerial": "disco",
+  "@writeChangesFallbackSerial": { "description": "Nombre predeterminado que mostrar para un disco sin uno de serie (poco probable)" },
```

Comment:
"sin un número de serie" ?

Reply:
I'm not sure if it is "nombre de fábrica" or "número de serie". All disks must have a serial number right? I honestly need more info on this one.

Reply:
Yes, all physical disks should have a serial number. Some virtualization managers appear to create virtual disks without a serial number though, hence the need for a fallback label.